integration-tests/generate_from_templates.py

```diff
@@ -101,5 +103,5 @@ def pathified(string):
 with open("Templates/users_template.yml.j2") as users:
     template = Template(users.read())
     with open(os.path.join(path, 'users.bobnet.yaml'), 'w') as f:
-        f.write(template.render())
+        f.write(template.render(auth_user=args.auth_user, auth_password=args.auth_password))
```

Comment:
You should also enable auth in `node.yaml`

Reply:
Auth is enabled by passing `vars(args)` into `template.render()` for node config. It is the same as explicitly listing every parameter that was parsed by parse_args() function, and as long as there is a parameter with the same name in a template it will be automatically substituded with an according value. And among arguments there is an `auth_type` argument that sets the auth type depending on value in the matrix.